src/app/modules/card-cluster/card-cluster.component.html

```diff
@@ -94,6 +94,9 @@
                   <fa-icon [icon]="card.nameIcon" *ngIf="card.nameIcon"></fa-icon>
                   {{ !card.name ? '&nbsp;' : card.name }}
                 </div>
+                <div *ngIf="card.tooltipModel" class="ds-text noPoint" [ngClass]="$any(card.className)" id="{{ card.id }}-name">
```

Comment:
any reason why we need to use `$any` here? would doing something like `{{card.className}}` work?